modules/ROOT/partials/partial_glossary.adoc

```diff
@@ -71,6 +71,22 @@ The smallest denomination of STRK. 1 fri = 10^-18^ STRK.
 
 A machine that runs the Starknet execution protocol. A full node stays synchronized with the latest state of the network, stores past blocks and transactions, and enables users to interact with the network, such as by querying its state or by broadcasting new transactions.
 
+[#l1]
+== Layer 1, L1
+
+The base layer of a blockchain network, such as Ethereum, where transactions are directly recorded and validated. It includes the underlying protocol and consensus mechanism that establish the foundational rules for the blockchain.
+
+[#l2]
+== Layer 2, L2
+
+Exists on top of an L1 blockchain, such as Ethereum, in order to add value to the L1, by providing features such as massive computation scale and security. Starknet is an L2 blockchain that exists on Ethereum.
```

Comment:
It doesn't bring security it scales ethereum without losing any security

Reply:
@LucasLvy 
I thought that scaling computation is the purpose of the rollup, but that the point of STARK proofs is to prove the validity of transactions, which sounds like a form of security, over and above what Ethereum provides.